pkgs/by-name/we/webfontkitgenerator/package.nix

```diff
@@ -69,9 +69,11 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   meta = with lib; {
-    description = "Webfont Kit Generator is a simple utility that allows you to generate woff, woff2 and the necessary CSS boilerplate from non-web font formats (otf & ttf)";
+    description = "Create @font-face kits easily";
+    longDescription = "Webfont Bundler is a simple utility that allows you to generate woff, woff2 and the necessary CSS boilerplate from non-web font formats (otf and ttf).";
+    changelog = "https://github.com/rafaelmardojai/webfont-kit-generator/releases/tag/${finalAttrs.version}";
     mainProgram = "webfontkitgenerator";
```

Comment:
Double checked upstream, and indeed it seems like the binary name did not change; that's somewhat confusing but we'll just have to live with it